### PR TITLE
i18n(hi): add `astro-pages.mdx`

### PR DESCRIPTION
**Pull request by at-the-vr**
#### Description (required)

- translates [Pages](https://docs.astro.build/hi/basics/astro-pages/) in hindi language
- minor change in nav.ts to match with title of Pages

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/hi/basics/astro-pages.mdx [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+import ReadMore from '~/components/ReadMore.astro';
+import Since from '~/components/Since.astro'
+
+**पेज** वे फ़ाइलें हैं जो आपके Astro परियोजना की `src/pages/` उपनिर्देशिका में रहती हैं। वे आपकी वेबसाइट के प्रत्येक पेज के लिए रूटिंग, डेटा लोडिंग और समग्र पेज लेआउट को संभालने के लिए जिम्मेदार हैं।
+
+## समर्थित पेज फ़ाइलें
+
+Astro `src/pages/` निर्देशिका में निम्नलिखित फ़ाइल प्रकारों का समर्थन करता है:
+- [`.astro`](#astro-pages)
```
Comment (at-the-vr) [inline src/content/docs/hi/basics/astro-pages.mdx:15]:
```suggestion
- [`.astro`](#astro-पेज)
```

File: src/content/docs/hi/basics/astro-pages.mdx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- [`.md`](#markdownmdx-pages)
```
Comment (at-the-vr) [inline src/content/docs/hi/basics/astro-pages.mdx:16]:
```suggestion
- [`.md`](#markdownmdx-पेज)
```

File: src/content/docs/hi/basics/astro-pages.mdx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+- `.mdx` ([MDX एकीकरण इंस्टॉल](/hi/guides/integrations-guide/mdx/#installation) होने के साथ)
+- [`.html`](#html-pages)
```
Comment (at-the-vr) [inline src/content/docs/hi/basics/astro-pages.mdx:18]:
```suggestion
- [`.html`](#html-पेज)
```

**Review by at-the-vr — COMMENTED** [review]
reviewing my own PR 😓

File: src/content/docs/hi/basics/astro-pages.mdx [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+**पेज** वे फ़ाइलें हैं जो आपके Astro परियोजना की `src/pages/` उपनिर्देशिका में रहती हैं। वे आपकी वेबसाइट के प्रत्येक पेज के लिए रूटिंग, डेटा लोडिंग और समग्र पेज लेआउट को संभालने के लिए जिम्मेदार हैं।
+
+## समर्थित पेज फ़ाइलें
+
+Astro `src/pages/` निर्देशिका में निम्नलिखित फ़ाइल प्रकारों का समर्थन करता है:
+- [`.astro`](#astro-पेज)
+- [`.md`](#markdownmdx-पेज)
+- `.mdx` ([MDX एकीकरण इंस्टॉल](/hi/guides/integrations-guide/mdx/#installation) होने के साथ)
+- [`.html`](#html-पेज)
+- `.js`/`.ts` (as [endpoints](/hi/guides/endpoints/))
```
Comment (Mayank-Malviya) [inline src/content/docs/hi/basics/astro-pages.mdx:19]:
```suggestion
- `.js`/`.ts` ([endpoints](/hi/guides/endpoints/) की तरह)
```

**Review by Mayank-Malviya — CHANGES_REQUESTED** [review]
leaving some suggestions, overall it seems good 👍

**Review by Mayank-Malviya — APPROVED** [review]
LGTM

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work 🙌

**Pull request merged**